### PR TITLE
[Chat] Fix long Seen display name as ellipsis

### PR DESCRIPTION
**Pull request by jimchou-dev**
# What

- Fix long Seen display name as ellipsis (truncate at 320px confirmed with UX)
image
- Set Chat sample to not show Participant Pane by default (1 liner fix)

# Why

Long Seen display name big https://skype.visualstudio.com/SPOOL/_workitems/edit/3240569
Participant Pane showing by default in Chat Sample https://skype.visualstudio.com/SPOOL/_workitems/edit/3240567

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by palatter — APPROVED** [review]
Looks good!

File: packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx [hunk 1]
```diff
@@ -147,7 +147,16 @@ export const ChatMessageActionFlyout = (props: ChatMessageActionFlyoutProps): JS
         subMenuProps: {
           id: 'chat-composite-message-contextual-menu-read-name-list',
           items: messageReadByList ?? [],
-          calloutProps: preventUnwantedDismissProps
+          calloutProps: preventUnwantedDismissProps,
+          styles: concatStyleSets({
```
Comment (palatter) [inline packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx:151]: I notice some places in the code they define the styles separately in their own variable?

**Review by JoshuaLai — APPROVED** [review]
Tested looks good to me

File: packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx (hunk 1 above)
Comment (jimchou-dev) [reply packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx:151]: In this same file they are also using concatStyle so for consistency I did the same here. And I think it make sense for certain styles that are used multiple places to be a variable.

**Pull request merged**